package.json

```diff
@@ -46,6 +46,7 @@
     "test:e2e": "turbo run test --filter=e2e-tests",
     "generate": "turbo run generate",
     "doc": "ts-node scripts/generate-docs.ts",
+    "changelog": "ts-node scripts/merge-changelogs.ts",
```

Comment:
[req] As disccussed check if the `after-bump` action i.e. related to the version script are still executed. 

[pp] Currently the `merge-changelogs.ts` is executed via the `changelog` scirpt above. Since we have still the after-bump scirpt I wonder if it would make sense to include that `merge-changelogs` in the after bump since there a lot of file adjustments are already done.